nle/env/base.py

```diff
@@ -214,7 +220,9 @@ def __init__(
 
         space_dict = {
             "glyphs": gym.spaces.Box(
-                low=0, high=nethack.MAX_GLYPH, **nethack.OBSERVATION_DESC["glyphs"]
+                low=0,
+                high=nethack.MAX_GLYPH,
+                **nethack.OBSERVATION_DESC["glyphs"],
```

Comment:
Wait, did black do this?

There are versions of Python that really really dislike the comma after `**kwargs`.

Reply:
I think, black did this when I run it with '-l 80'. Right now black uses the default option for line width.

Reply:
But the comma after the `**kwargs` arg?

See https://github.com/psf/black/issues/419 -- what version of `black` are you using? I am not sure we should have commas there, and at any rate there's no need for this PR to change these lines.